lapis/cli/simulate.py

```diff
@@ -38,20 +54,26 @@ def makePickle(self, record):
 @click.option("--until", type=float)
 @click.option("--log-tcp", "log_tcp", is_flag=True)
 @click.option("--log-file", "log_file", type=click.File("w"))
+@click.option("--log-telegraf", "log_telegraf", is_flag=True)
 @click.pass_context
-def cli(ctx, seed, until, log_tcp, log_file):
+def cli(ctx, seed, until, log_tcp, log_file, log_telegraf):
     ctx.ensure_object(dict)
     ctx.obj['seed'] = seed
     ctx.obj['until'] = until
     if log_tcp:
-        socketHandler = JSONSocketHandler(
+        socketHandler = LoggingSocketHandler(
             'localhost', logging.handlers.DEFAULT_TCP_LOGGING_PORT)
```

Comment:
It might be worth being able to override the host and port. Is it possible to use ``--log-tcp`` (and telegraf) both as a flag and as an option?

* ``--log-tcp`` enables logging to ``localhost`` on the standard TCP port
* ``--log-tcp foohost 12312`` enables logging to ``foohost`` on port ``12312``.

Reply:
I opened a new issue. I planned to enable this via configuration, but an option for CLI is also a good idea.